README.Rmd

```diff
@@ -0,0 +1,74 @@
+---
+output: github_document
+bibliography: doc/default_prediction_refs.bib
+---
+
+# Credit Default Payment Predictor
+- Authors: Arushi Ahuja, Karanpreet Kaur, Lianna Hovhannisyan, Taiwo Owoseni
+
+Using Logistic Regression to predict whether the customer will do credit default payment next month or not.
+
+## About
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+In this project, we built a classification model using Logistic Regression to predict if the credit account holders will make a default payment or not next month. The model was trained on features that hold information about the client's last 6 months bill and payment history, as well as several characteristics: age, marital status, gender, etc. Overall, we are more interested in minimizing Type | error (predicting not default payment, when in reality client made a default payment next month) and Type || error (predicting default payment, when in reality client did not do default next month), we are using primarily $f1$ as scoring metric. Our model performed fairly well....
```

Comment:
Changed the grammatical errors in capital:

In this project, we built a classification model using Logistic Regression to predict if [NO THE] credit account holders will make a default payment [NO OR NOT] next month. The model was trained on features that hold information about the client's last 6 months bill and payment history, as well as several OTHER characteristics SUCH AS: age, marital status, AND gender[NO ETC]. Overall, we are more interested in minimizing Type | error (predicting NO default payment, when in reality THE client made a default payment THE FOLLOWING MONTH), AS OPPOSED TO Type || error (predicting default payment, when in reality NO DEFAULT PAYMENT WAS MADE BY THE CLIENT), we are using [NO PRIMARILY] $f1$ as OUR PRIMARY scoring metric. Our model performed fairly well....